src/main/java/uk/gov/hmcts/reform/sendletter/tasks/reports/EmailSender.java

```diff
@@ -0,0 +1,75 @@
+package uk.gov.hmcts.reform.sendletter.tasks.reports;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.stereotype.Component;
+
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+
+@Component
+@ConditionalOnProperty(prefix = "spring.mail", name = "host")
+public class EmailSender {
+
+    private static final Logger log = LoggerFactory.getLogger(EmailSender.class);
+
+    static final String EMAIL_BODY = "This is an auto generated email. Do not respond to it.";
+
+    private final JavaMailSender mailSender;
+    private final String from;
+
+    public EmailSender(
```

Comment:
you have multiple send method which would be more useful if this was a library. May be just have one which we are going to use for now ?

Reply:
I now. I was more hoping for general review and this being a non-breaking summary which I can easily remove. Plus will give me room to do an exception unit test case as well as it is left behind